src/frontend/src/services/ApplianceRateService.js

```diff
@@ -1,10 +1,21 @@
 import { ErrorHandler } from "@/Helpers/ErrorHandler"
+import { Paginator } from "@/Helpers/Paginator"
+import { convertObjectKeysToCamelCase } from "@/Helpers/Utils"
+import { resources } from "@/resources"
 
 import ApplianceRateRepository from "@/repositories/ApplianceRateRepository"
 
 export class ApplianceRateService {
-  constructor() {
+  constructor(appliancePersonId = null) {
```

Comment:
Should we allow this to be instanciated with `null` ? Or just just make it mandatory like in `ApplianceLogService`

Reply:
I updated this. Figured it wasn't being used without the `appliancePersonId` anyway.